source/SpinalHDL/Getting Started/VSCodium.rst

```diff
@@ -18,11 +19,9 @@ corner: "Multiple build definitions found. Which would you like to use?". Click
 Wait while running ``sbt bloopInstall``. Then a warning pop-up appears, you can
 ignore it (don't show again).
 
-In the explorer, find and open ``MyTopLevel.scala``. Wait a little bit, and see
-the ``run | debug`` line that is displayed by Metals, before each ``App``. For
-instance, click on ``run`` just above ``object MyTopLevelVerilog``. It performs
+In the search bar, find and open ``MyTopLevel.scala``.  Once it loads select Menu Bar -> Run -> Run Without Debugging.  It performs
```

Comment:
I don't get why you'd go to the search bar? While you could find the file there by searching some string from the file doesn't it make more sense to go to the explorer and open `hw/spinal/projectname/MyTopLevel.scala`?

Tried metals now on 3 machines, on all three I get the `run | debug` buttons - I think we can leave that in the docs.

Reply:
That is because you know where the files are located.  The new user does not know where the files are located, and so has to do a search.   I have several times had to do  

find . -name file.name

Much easier to save the new user that step.

Reply:
yes, much easier to tell them the path I'd think?